Renon hybrid IFLXX

sirhaplo opened the pull request:
New definition for Renon Hybrid IFL06. 
Should work for different sizes too.

Data is extracted from solarman app and tested with local setup

Proof of work :
![image](https://github.com/user-attachments/assets/bd71fe25-3d92-4dfe-b629-d30ac12187c7)
![image](https://github.com/user-attachments/assets/a3da3b8f-a499-4629-84ed-8187ebc0d032)

davidrapan commented:
Hi! I also added your file to my integration (hope you don't mind) [davidrapan/ha-solarman/renon_ifl.yaml](https://github.com/davidrapan/ha-solarman/blob/main/custom_components/solarman/inverter_definitions/renon_ifl.yaml). Can I ask why there are missing values for Load L3 (C) when Grid L3 (C) does exist? Is that correct and it's not Three Phase?

sirhaplo commented:
> Hi! I also added your file to my integration (hope you don't mind) [davidrapan/ha-solarman/renon_ifl.yaml](https://github.com/davidrapan/ha-solarman/blob/main/custom_components/solarman/inverter_definitions/renon_ifl.yaml). Can I ask why there are missing values for Load L3 (C) when Grid L3 (C) does exist? Is that correct and it's not Three Phase?

Be my guest, but keep in mind that is a partial configuration as you saw.
I added only registers that i could test ( single phase domestic line ), or that were obvious by reverse engeenering. 
It still miss some values

davidrapan commented:
Sure thing! 👍

davidrapan commented:
If you have any further edits in the future, I'm open to PR! (This repo is not that active which is one of many reasons I decided to recreate the integration)
